src/components/molecules/SectionRenderer/SectionHeader.tsx

```diff
@@ -35,7 +35,12 @@ function SectionHeader(props: SectionHeaderProps) {
     expandSection,
     collapseSection,
   } = props;
+
+  const {customization = {}} = sectionBlueprint;
```

Comment:
could you revert the changes on this file? I remember it's not needed anymore and I'm currently refactoring the section blueprint components in a separate branch